public_html/style.css

```diff
@@ -0,0 +1,255 @@
+/* Common style css */
+:root {
+
+    /* color */
+    --color-primary: #177d89;
+    --color-secondary: #f2c833;
+    --color-black: #000000;
+    --color-black-1e: #1e1906;
+    --color-black-gray: #3f3f3f;
+    --color-white: #fff;
+    --color-line: #4597a1;
```

Comment:
I don't know about this color, whether you use it or not.

Reply:
I have used all color you have seen in root